src/olympia/zadmin/admin.py

```diff
@@ -40,6 +44,21 @@ def related_single_content_link(obj, related_field):
         return ''
 
 
+# Hijack the admin's login to use our pages.
```

Comment:
Just wondering what this change is about. It doesn't seem to be mentioned at https://github.com/mozilla/addons-server/issues/15292.

Reply:
I just moved the code out of urls.py (it was an odd place to have it - I should have moved it previously)